### ISSUE TITLE
Good reference for toxicity of aniline-containing drugs?

### ISSUE THREAD

**Issue by mattodd** [OpenSourceMalaria/OSMSeries4Paper1#48]
What's the problem with aniline-containing drugs? 

Need this for the [paper](https://docs.google.com/document/d/1vlOHNo9bxmx24KmNdJz5ZmjXyVP9uZoHJcnGj181NZE/edit?usp=sharing) (p9), as an explanation for why some potent compounds (e.g. MMV1581334) were not further pursued. @drc007 (#46) suggested https://europepmc.org/article/med/6990249, from 1980.

Any others (maybe more recent or bigger studies)?

(This is one of those well-understood-community-rules-of-thumb-hard-to-reference things).

**Comment by MedChemProf** [comment]
@mattodd I edited the section that included the aniline analog again to include some additional discussion as well as updated references in regards to aniline reactive metabolites. I include the below three aniline literature references in the manuscript:

•	Antonia F. Stepan, A.F., Walker, D.P., Bauman, J., Price, D.A., Baillie, T.A., Kalgutkar, A.S., Aleo, M.D., “Structural Alert/Reactive Metabolite Concept as Applied in Medicinal Chemistry to Mitigate the Risk of Idiosyncratic Drug Toxicity: A Perspective Based on the Critical Examination of Trends in the Top 200 Drugs Marketed in the United States”, Chem. Res. Toxicol. (2011) 24(9), pp. 1345-1410. https://doi.org/10.1021/tx200168d
•	Amit S. Kalgutkar, “Should the Incorporation of Structural Alerts be Restricted in Drug Design? An Analysis of Structure-Toxicity Trends with Aniline-Based Drugs”, Current Medicinal Chemistry (2015) 22, pp. 438-464. https://doi.org/10.2174/0929867321666141112122118 
•	Taylor M. Sodano, Logan A. Combee, and Corey R. J. Stephenson, “Recent Advances and Outlook for the Isosteric Replacement of Anilines”, ACS Med. Chem. Lett. (2020) 11, pp. 1785-1788. https://dx.doi.org/10.1021/acsmedchemlett.9b00687 

Beyond the references listed above, some additional general references on reactive metabolites are listed below (aniline is one of the topics covered in the references.)

•	Amit S. Kalgutkar, Iain Gardner, R. Scott Obach, Christopher L. Shaffer, Ernesto Callegari, Kirk R. Henne, Abdul E. Mutlib, Deepak K. Dalvie, Jae S. Lee, Yasuhiro Nakai, John P. O'Donnell, Jason Boer, Shawn P. Harriman, “A comprehensive listing of bioactivation pathways of organic functional groups”, Curr. Drug Metab. (2005) 6(3), pp. 161-225. https://doi.org/10.2174/1389200054021799
•	Suvi T. M. Orr, Sharon L. Ripp, T. Eric Ballard, Jaclyn L. Henderson, Dennis O. Scott, R. Scott Obach, Hao Sun, and Amit S. Kalgutkar, “Mechanism-Based Inactivation (MBI) of Cytochrome P450 Enzymes: Structure–Activity Relationships and Discovery Strategies To Mitigate Drug–Drug Interaction Risks”, J. Med. Chem. (2012) 55(11), pp. 4896-4933. https://doi.org/10.1021/jm300065h
•	Kalgutkar, A.S. and Dalvi, D., “Predicting Toxicities of Reactive Metabolite–Positive Drug Candidates”, Annu. Rev. Pharmacol. Toxicol. (2015) 55, pp. 35-54. https://doi.org/10.1146/annurev-pharmtox-010814-124720

**Comment by mattodd** [comment]
Hi @MedChemProf - I noticed this new paper about mutagenicity of amines (and, interestingly, what can be done about it)

https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00514

In your view, is it worth a citation, to make our point that anilines may not necessarily be a terrible idea?

**Comment by MedChemProf** [comment]
Actually, the chloro-aniline MMV1581334 (along with some of the analogs around it) were very intriguing and we may have been scared off too easily. Possibly a combination of some sort of aniline analog in the northeast region of the molecule in conjunction with the appropriate benzylic substitution on the linker may have resulted in some very potent compounds. I added a brief statement along these lines to the text discussing the SAR in the current Figure 5. I also added the new reference to the as part of the updated discussion. The reference is an ASAP article right now, but by time we get the manuscript completed, we can update the reference details.
